app/values/hls_manifest.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class HlsManifest
```

Comment:
Can we have class-level documentation here to clarify how and why it's different from the HlsManifest::Primary?

Reply:
Added some docs.

Reply:
thank you